test/e2e/clusterctl_upgrade_test.go

```diff
@@ -241,8 +241,8 @@ var _ = Describe("When testing clusterctl upgrades using ClusterClass (v1.7=>cur
 			InitWithInfrastructureProviders: []string{fmt.Sprintf(providerDockerPrefix, stableRelease)},
 			InitWithProvidersContract:       "v1beta1",
 			// Note: Both InitWithKubernetesVersion and WorkloadKubernetesVersion should be the highest mgmt cluster version supported by the source Cluster API version.
-			InitWithKubernetesVersion:   "v1.30.0",
-			WorkloadKubernetesVersion:   "v1.30.0",
+			InitWithKubernetesVersion:   "v1.31.4",
+			WorkloadKubernetesVersion:   "v1.31.4",
```

Comment:
The mapper of CAPI 1.8.x does not have v1.31.4 (it has v1.31.0)

Ideally we would have used v1.31.0 so there is no risk that CAPD picks a wrong version of the v1.31.4 image (which is only in the mapper on main and also might be overwritten later)

Reply:
fixed in https://github.com/kubernetes-sigs/cluster-api/pull/11659 + added a comment so the next time I will remember to check versions supported by the kind mapper

Reply:
Yup good point, thx!